### PR TITLE
Rewrite of conda naming conventions

### PR DESCRIPTION
**Pull request by ingkebil**
Update and rewrite of the how to name the conda environments

How to view changes made to manual?

```
git clone -b conda https://github.com/Clinical-Genomics/development.git
cd development
conda create -n D_mkdocs
source activate D_mkdocs
pip install -r requirements.txt
mkdocs serve
```

go to your browser: http://127.0.0.1:8000

### CODE REVIEW

File: docs/conda/conda_conventions.md [hunk 1]
```diff
@@ -1,15 +1,38 @@
...
+   - creation_date: `%y%m%d`
+   - timestamp: `%y%m%d %H%M%S`.
+   - name: whatever makes sense.
+   - signature: showing who created the environment. Use the two- or three letter name acronyms assigned to you.
+
+## Types of environments
+
+As a developer your main kind of environment will be the development conda environment which is prefixed with a `D_`.
+
+Both the stage and production environments should already be created.
```
Comment (patrikgrenfeldt) [inline docs/conda/conda_conventions.md]: Why should P- and S- environments already be created? I would think I would create D-usalt first and when I think I'm ready for production I create S-usalt and when I'm sure I'm ready for production I'd create P-usalt?

Comment (ingkebil) [reply docs/conda/conda_conventions.md]: Sure, this process needs to be fleshed out better. Let's remove this whole chapter of this page and move it to another PR

**Comment by ingkebil** [comment]
If the changes are acceptable, could you approve @patrikgrenfeldt @sylvinite ?

**Comment by ingkebil** [comment]
I will merge this without your explicit approval

**Pull request merged**